README.md

```diff
@@ -6,13 +6,115 @@
   Plone Content Export and Import
 </h1>
 
-Package supporting the export and import of content, principals, relations, and translations from and to a Plone site.
+Package supporting the export and import of content, principals, relations, translations, discussions, and redirects from and to a Plone site.
 
 ## Introduction
 
 This package is a slimmer version of the awesome [collective.exportimport](https://github.com/collective/collective.exportimport).
 
-While `collective.exportimport` supports older Plone versions and Python 2, and also takes care of data conversion -- i.e.: from Archetypes to Dexterity -- this package focus only on latest Plone and Python.
+While `collective.exportimport` supports older Plone versions and Python 2, and also takes care of data conversion from Archetypes to Dexterity, this package focus only on latest Plone and Python.
+
+
+## Documentation
+
+[`plone.exportimport` documentation](https://6.docs.plone.org/admin-guide/export-import.html)
+
+
+
+## Installation
+
+> [!IMPORTANT]
+> This package supports sites running Plone version 6.0 and above.
+
+> [!IMPORTANT]
+> This package is now included with Plone 6.1 and above by default.
+> The following installation instructions are relevant only for Plone 6.0 sites.
+
+Add `plone.exportimport` to the Plone installation using `pip`.
```

Comment:
As usual, this depends. :-/  This package is *not* a dependency of `Products.CMFPlone`, so it may be be included, even if you are on Plone 6.1.  It is a dependency of `plone.distribution`, which itself is a dependency of `plone.volto` and `plone.classicui`, both of which are dependencies of the `Plone` package.

So maybe rewrite to something like this:

```suggestion
> This package is now included with Plone 6.1 and above by default.

If `plone.exportimport` is not yet available in your Plone installation, add it using `pip`.
```